agents/agent_engine/tutorial_get_started_with_code_execution.ipynb

```
@@ -0,0 +1,2072 @@
+        "id": "5wvvD1uf8agN"
+      },
+      "source": [
+        "The sandbox can return multiple types of outputs, and we need to handle each appropriately.\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "WEU_nnbk8cMx"
+      },
```

Ruff (0.14.3)

`open()` should be replaced by `Path.open()`

(PTH123)